cl_manycore/libraries/bsg_manycore_cuda.cpp

```diff
@@ -458,6 +472,49 @@ int hb_mc_tile_group_enqueue (	hb_mc_device_t* device,
 
 
 
+/**
+ * Takes in a kernel name, argument count and list of arguments 
+ * and initializes a hb_mc_kernel_t struct for the tile group.
+ * @param[in]  tg            Pointer to tile group. 
+ * @param[in]  name          Kernel name that is to be executed on tile group
+ * @param[in]  argc          Number of input arguments to the kernel
+ * @param[in]  argv          List of input arguments to the kernel
+ * @return HB_MC_SUCCESS on success and HB_MC_FAIL on failure. 
```

Comment:
Technically that's not accurate. It can return HB_MC_NOMEM. I typically copy and paste something like this when I'm too lazy to list all the possible return codes.
```c
/** 
 * ...
 * @return HB_MC_SUCCESS on success. Otherwise an error code defined in bsg_manycore_errno.h.`
*/

Reply:
Ahhh you're right, I copy pasted without paying attention, will fix it!